src/operator/src/statement/copy_table_from.rs

```diff
@@ -427,6 +432,10 @@ impl StatementExecutor {
                     rows_inserted += rows;
                     insert_cost += cost;
                 }
+
+                if rows_inserted >= max_insert_rows {
```

Comment:
hi @v0y4g3r , should we immediately stop when rows_inserted reaches max_insert_rows?

Reply:
> hi @v0y4g3r , should we immediately stop when rows_inserted reaches max_insert_rows?

Maybe we should break the outer loop and terminate the execution of whole `COPY` statement.

Reply:
I suggest instead we extract a function to return `(rows_inserted, insert_cost)` and return here. break with label is generally hard to interpret and the logic here is large enough to have a dedicated function.